source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst

```diff
@@ -8,7 +8,63 @@ Drive Tab
 
 .. image:: images/using-the-labview-dashboard-with-c++-java-code/drive-tab.png
 
-Most of the indicators on the Drive tab utilize arrays.
+The :guilabel:`Select Autonomous...` dropdown can be used so show the available autonomous routines and choose one to run for the match.
+
+.. tabs::
+
+    .. code-tab:: java
+
+       SmartDashboard.putStringArray("Auto List", ["Drive Forwards", "Drive Backwards", "Shoot"]);
```

Comment:
Lists don't use square braces in Java. I don't know the correct way to do this though.